src/tests/ov_helpers/ov_models/src/activation.cpp

```diff
@@ -63,7 +63,7 @@ std::shared_ptr<ov::Node> makeActivation(const ov::Output<Node>& in,
     case ov::test::utils::ActivationTypes::Relu:
         return std::make_shared<ov::op::v0::Relu>(in);
     case ov::test::utils::ActivationTypes::LeakyRelu: {
-        auto leaky_slope = std::make_shared<ov::op::v0::Constant>(ov::element::f32, inShape, constantsValue);
```

Comment:
@itikhono @iefode please, review this file

Reply:
LGTM